docs/tcpAgent.md

```diff
@@ -0,0 +1,140 @@
+# Launching inbound TCP agents
+Jenkins provides a number of ways of connecting remote agents.
+Two of the most popular are outbound SSH agents and inbound TCP agents.
+SSH agents, most commonly used on Unix platforms, are master initiated.
+The master creates the connection when it needs.
+Inbound TCP agents, most commonly used on Windows platforms, are agent initiated.
+The agent must first connect to the master and then the master sends commands as needed.
+These were formerly known as JNLP agents, but that name was erroneous and confusing.
+This document describes some of the primary mechanisms for launching inbound TCP agents.
+
+For additional information about Jenkins agents see [Distributed builds](https://wiki.jenkins.io/display/JENKINS/Distributed+builds#Distributedbuilds-HavemasterlaunchagentonWindows).
+
+## Launch mechanisms
+Part of the agent status page for TCP agents looks something like this:
+![Tcp agent status UI](tcpAgentStatus.jpg)
+This shows a couple of launch methods, but there are other mechanisms.
+If you use a different mechanism than clicking the "Launch" button, this page contains parameters that you will need. 
+The long string of hex digits is a secret key that the client needs to establish the connection. 
+Depending on how you connect you may also need the agent name, "AgentName" in this example, or the (misnamed) jnlpUrl, "https://myjenkins.example.com/computer/AgentName/slave-agent.jnlp".
+
+### "Launch" button
+This is the only launch mechanism that actually uses JNLP.
+When properly configured and supported, clicking this button causes a JNLP file to be downloaded via the web browser and launched with Java WebStart. 
+
+While this is the simplest mechanism it has many drawbacks.
+This is an outdated technology that is being phased out.
```

Comment:
These three lines read like bullet points. Do you want to make them bullets?

Reply:
That's a possibility, but I don't think it lays out / reads as well. It might work better in a system with richer formatting. Especially since the third point spans multiple paragraphs.